Ws-Peroth/20211007/MapGeneratorBSP.cs

```diff
@@ -0,0 +1,251 @@
+﻿using UnityEngine;
+
+public class MapGeneratorBSP : MapInitializer
+{
+    // 각 구역별 border구역 크기
+    private readonly int _borderSizeX = 2;
```

Comment:
```suggestion
    private cont int BorderSizeX = 2;
```
readonly와 const의 차이점을 짚어보면 좋을 것 같습니다.
readonly는 class, const는 일반적인 value type과 struct 등에 이용합니다.
또한, readonly를 쓸 때는 static을 붙여주는게 메모리에서 유리하고 readonly/const를 이용할 때는 변수명을 PascalCase로 짓습니다.